pkg/cimfs/mount_cim.go

```diff
@@ -15,6 +15,10 @@ import (
 	"golang.org/x/sys/windows"
 )
 
+const (
```

Comment:
nit: I only see this code being moved up a bit. If there are no other changes to this file can we remove these edits?